apps/web/src/pages/Applications/ApplicationQuestionsPage/ApplicationQuestionsPage.tsx

```diff
@@ -66,9 +76,166 @@ export const getPageInfo: GetApplicationPageInfo = ({
+              description:
+                "Message displayed to users when saving screening question responses is successful.",
+            }),
+          );
+          navigate(paths.applicationReview(application.id));
+        }
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Error: adding experience failed",
+            id: "moKAQP",
+            description:
+              "Message displayed to user after experience fails to be created.",
+          }),
+        );
+      });
+  };
+
+  return (
+    <>
+      <div
+        data-h2-display="p-tablet(flex)"
+        data-h2-align-items="p-tablet(flex-end)"
+        data-h2-justify-content="p-tablet(space-between)"
+        data-h2-margin-bottom="base(x1)"
+      >
+        <Heading data-h2-margin="base(0)" data-h2-line-height="base(1)">
+          {pageInfo.title}
+        </Heading>
+        <Link
+          mode="inline"
+          type="button"
+          color="secondary"
+          href={paths.applicationQuestionsIntro(application.id)}
+        >
+          {intl.formatMessage({
+            defaultMessage: "Review instructions",
+            id: "VcpIlx",
+            description:
+              "Link text to return to an introduction page on an application",
+          })}
+        </Link>
+      </div>
+      <p data-h2-margin="base(x1, 0)">
+        {intl.formatMessage({
+          defaultMessage:
+            'Don\'t forget to take a break if you need to! Using the "Save and quit for now" button, you can record your progress and return to questions you might be stuck on later.',
+          id: "tMnjyJ",
+          description:
+            "Notice that application can be saved and returned to at a later time.",
+        })}
+      </p>
+      <BasicForm
+        onSubmit={handleSubmit}
+        options={{
+          defaultValues: dataToFormValues(
+            screeningQuestions,
+            screeningQuestionResponses,
+          ),
+        }}
+      >
+        {screeningQuestions.length ? (
+          screeningQuestions.map((question, index) => (
+            <React.Fragment key={question.id}>
+              <Heading level="h3" size="h4">
+                {intl.formatMessage(
+                  {
+                    defaultMessage: "Question {number}",
+                    id: "/sBGov",
+                    description: "Heading for a specific screening question",
+                  },
+                  { number: index + 1 },
+                )}
+              </Heading>
+              <input type="hidden" name={`answers.${index}.id`} />
+              <input type="hidden" name={`answers.${index}.questionId`} />
+              <TextArea
+                id={`answers.${index}.answer`}
+                name={`answers.${index}.answer`}
+                label={getLocalizedName(question.question, intl)}
+                rules={{ required: intl.formatMessage(errorMessages.required) }}
+              />
```

Comment:
Don't forget the word limit and placeholder text (whatever it's supposed to be :laughing:).
![image](https://user-images.githubusercontent.com/8978655/233648682-d47d27e7-61d8-46b3-acff-1fc78d94cbce.png)

Reply:
Oops, I got the word counter part while we wait for the placeholder text.

[add word limit to screening question answer](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6311/commits/368ecf55bf2945ee5349dc65169dc1ef231f9deb)